vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -76,4 +76,8 @@ void AccessibilityInfo::getRecommendedTimeoutMillis(
   onSuccess(mSec);
 }
 
+void AccessibilityInfo::isAccessibilityServiceEnabled(std::function<void(bool)> const & onSuccess) {
+  // no-op
```

Comment:
This is expected to return something (either true or false) via the callback that is passed in. If we want to add a stub, it should return one of those two, or an error.

What does the implementation for this look like on other platforms?

Reply:
Sure I can check it out.